data/json/items/tools.json

```diff
@@ -257,26 +257,25 @@
     "type": "GENERIC",
     "category": "tools",
     "name": "atomic coffee maker",
-    "description": "Never sacrifice taste for convenience, when you can have both with the Rivtech atomic coffee maker!  Its simple and robust atomic-age construction guarantees a service life of at least 160 million years.",
+    "description": "This is a Curie-G coffeemaker, by CuppaTech.  It famously uses a radioactive generator to heat water for coffee. Normally the water is heated using energy stored in a capacitor, and makes ordinary coffee.  However, as a special feature, water from the RTG containment area can be used, giving the coffee a very special kick.  The Curie-G is illegal in most countries.",
     "weight": 4102,
-    "volume": 3,
+    "volume": "1000 mL",
     "price": 100000,
     "to_hit": -2,
     "bashing": 5,
     "material": [ "plastic", "aluminum" ],
     "symbol": ",",
     "color": "light_green",
-    "qualities": [ [ "BOIL", 1 ] ],
```

Comment:
Is removing BOIL quality intended? It's vital for this item to be able to make coffee without any external container.

Reply:
Right, I forgot that it wasn't a use effect. I'll put that back.